deployment/ansible/roles/raster-foundry.docker/tasks/main.yml

```diff
@@ -6,6 +6,4 @@
   user: name="{{ item }}"
         groups=docker
         append=yes
-  with_items:
```

Comment:
We could also just move the addition of the Jenkins user to the Docker group over to the Jenkins role. That would help isolate issues related to Jenkins from appearing in local development setups.